menu/visual.cpp

```diff
@@ -92,7 +92,12 @@ void anim::snakeGamemodes() {
 }
 void anim::snakeSettings() {
     std::cout << Term::move_cursor(7, 10) << Term::color24_fg(255, 255, 0)
-              << "SETTINGS!" << Term::color(Term::fg::reset) << std::flush;
+              << "SETTINGS!" << Term::move_cursor(9, 6)
+              << Term::color24_fg(21, 96, 189) << "Controls"
+              << Term::move_cursor(10, 6) << "Theme" << Term::move_cursor(11, 6)
+              << "Lorem ipsum" << Term::move_cursor(9, 17)
```

Comment:
Place holder text

Reply:
Yes, I did that so that I have three different options to choose from. Should I remove it?

Reply:
I mean will we add another option there?

Reply:
But I guess it's fine foe now ^^

Reply:
Probably. Maybe sound or smth.